{{cookiecutter.project_dash_slug}}/.circleci/config.yml

```diff
@@ -47,8 +47,11 @@ default: &steps
                 }
             EOF
             mkdir test-reports
-            pipenv run python -m xmlrunner discover -s tests -t . -p '*.py'
-            ls -1 TEST-*.xml | awk -F. -f move-results.awk | bash
+            save() {
+              ls -1 TEST-*.xml | awk -F. -f move-results.awk | bash
+            }
+            pipenv run python -m xmlrunner discover -s tests -t . -p '*.py' || (save && false)
+            save
```

Comment:
This is pretty opaque. Please make it simpler.

Reply:
@jeremycarroll can you help me understand what this code is supposed to do? I don't know awk very well.

Reply:
I think I should recode in python